packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -344,6 +425,11 @@ class NavigatorState extends State<Navigator> {
       _pop();
   }
 
+  /// Whether this navigator can be popped.
+  ///
+  /// The initial route cannot be popped off the navigator, which implies that
+  /// this function returns true only if popping the navigator would not remove
+  /// the initial route.
```

Comment:
That sentence makes no sense. "X, therefore this function returns true only if X."